src/plugins/intel_gpu/tests/test_cases/scatter_nd_update_gpu_test.cpp

```diff
@@ -12,17 +14,266 @@
+                                                          ov::Shape(updates_vec.begin(), updates_vec.end()));
+
+        for (size_t i = 0; i < outputs_ref.size(); ++i) {
+            EXPECT_EQ(outputs_ref[i], outputs_ptr[i]);
+        }
+    }
+};
+
+TEST_P(scatter_nd_update_random_test, random)
+{
+    auto param = GetParam();
+    if(param.input_type == data_types::u8)
+        this->execute<uint8_t, uint8_t>(param);
+    else if (param.input_type == data_types::i8)
+        this->execute<int8_t, int8_t>(param);
+    else if (param.input_type == data_types::i32)
+        this->execute<int32_t, int32_t>(param);
+    else if (param.input_type == data_types::i64)
+        this->execute<int64_t, int64_t>(param);
+    else if (param.input_type == data_types::f16)
+        this->execute<FLOAT16, uint16_t>(param);
+    else if (param.input_type == data_types::f32)
+        this->execute<float, float>(param);
+    else
+        std::cout << "unidentified data type" << std::endl;
+}
+
+INSTANTIATE_TEST_SUITE_P(scatter_nd_update_gpu_random_test_bsv32_fsv16_fp32, 
+                         scatter_nd_update_random_test,
+                         testing::ValuesIn( 
+                             std::vector<scatter_nd_update_basic_test_params>{
+                             { data_types::f32, data_types::f32, data_types::f32, 
+                               format::bfyx, format::bfyx, format::bfyx,
+                               format::bs_fs_yx_bsv32_fsv16, format::bs_fs_yx_bsv32_fsv16, format::bs_fs_yx_bsv32_fsv16,
+                               {48, 24, 3, 3}, {3, 2, 1, 1}, {3, 3, 1, 3},
+                               2 } 
+                         }));
+
+INSTANTIATE_TEST_SUITE_P(scatter_nd_update_gpu_random_test_fsv16_fp32_yx, 
+                         scatter_nd_update_random_test,
+                         testing::ValuesIn( 
+                             std::vector<scatter_nd_update_basic_test_params>{
+                             { data_types::f32, data_types::f32, data_types::f32, 
+                               format::bfyx, format::bfyx, format::bfyx,
+                               format::b_fs_yx_fsv16, format::b_fs_yx_fsv16, format::b_fs_yx_fsv16,
+                               {48, 24, 3, 3}, {3, 2, 1, 1}, {3, 3, 1, 3},
+                               2 } 
+                         }));
+
+INSTANTIATE_TEST_SUITE_P(scatter_nd_update_gpu_random_test_fsv16_fp32_zyx, 
+                         scatter_nd_update_random_test,
+                         testing::ValuesIn( 
+                             std::vector<scatter_nd_update_basic_test_params>{
+                             { data_types::f32, data_types::f32, data_types::f32, 
+                               format::bfzyx, format::bfyx, format::bfzyx,
+                               format::b_fs_zyx_fsv16, format::b_fs_zyx_fsv16, format::b_fs_zyx_fsv16,
+                               {48, 24, 3, 3, 10}, {5, 2, 1, 1}, {5, 10, 1, 3, 3},
+                               2 } 
```

Comment:
I am seeing all of the indices ranks are 2. Could you add differnt test for 3 or 4?